zulip_bots/zulip_bots/bots/ping/doc.md

```diff
@@ -0,0 +1,29 @@
+# Ping Bot
+
+This bot allows users to check availability of a given website or IP
+address across 10 random locations using the free API provided by
+[check-host.net](https://check-host.net). The API documentation can be
+found [here](https://check-host.net/about/api).
+
```

Comment:
Can you add a new section "Setup"? If this bot doesn't require any specific setup, just add the section
```
This bot does not require any special setup. Just run it according to the
instructions in [this guide](https://zulipchat.com/api/running-bots#running-a-bot).
```
(taken from tictactoe).